library/rsa.c

```diff
@@ -1550,7 +1550,12 @@ int mbedtls_rsa_rsassa_pss_sign( mbedtls_rsa_context *ctx,
         return( MBEDTLS_ERR_RSA_BAD_INPUT_DATA );
 
     hlen = mbedtls_md_get_size( md_info );
-    slen = hlen;
+
+    /* Align with FIPS-186-4 */
```

Comment:
Could you expand this comment to explain how this aligns with FIPS 186-4? Please refer to sections within the document.

Reply:
Sure

Reply:
I've expanded the comment.